user_docs/en/changes.t2t

```diff
@@ -10,12 +10,12 @@ What's New in NVDA
 
 
 == Changes for Developers ==
-- Updated pySerial to version 3.4
+- Updated pySerial to version 3.4.
 - the validate module is now only available from configobj. Code should now do from configobj import validate rather than import validate.
-- textInfos.Point and textInfos.Rect are removed and replaced by locationHelper.Point and locationHelper.RectLTRB, respectively 
+- textInfos.Point and textInfos.Rect are removed and replaced by locationHelper.Point and locationHelper.RectLTRB, respectively.
 - braille.BrailleHandler._get/set_tether have been removed. use getTether and setTether respectively.
-- config.getConfigDirs has been removed
-- config.ConfigManager.getConfigValidationParameter is removed. A bug in getConfigValidation has been fixed in order for it being able to fully replace getConfigValidationParameter
+- config.getConfigDirs has been removed.
+- config.ConfigManager.getConfigValidationParameter is removed. A bug in getConfigValidation has been fixed in order for it being able to fully replace getConfigValidationParameter.

```

Comment:
Can this line possibly be made more clear? Something like this, perhaps?

```suggestion
- config.ConfigManager.getConfigValidationParameter has been removed. A bug in getConfigValidation was fixed in order for it to be able to fully replace getConfigValidationParameter.
```

Reply:
Even better, you can remove the words "in order" from that sentence.